internal/storage/database/repositories/users_db/users_db.go

```diff
@@ -0,0 +1,86 @@
+package users_db
+
+import (
+	"booker/internal/lib/api/models"
+	"booker/internal/storage/database"
+	"context"
+	"errors"
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgconn"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"log/slog"
+)
+
+var ErrEmailAlreadyExists = errors.New("Пользователь с email уже существует. ")
+var ErrUserNotFound = errors.New("Пользователь не найден ")
```

Comment:
```suggestion
var ErrEmailAlreadyExists = errors.New("Пользователь с email уже существует.")
var ErrUserNotFound = errors.New("Пользователь не найден.")
```

Когда будешь это читать, то открой форточку